tools/build_defs/cmake.bzl

```diff
@@ -77,6 +77,7 @@ def _create_configure_script(configureParameters):
         dict(ctx.attr.cache_entries),
         dict(ctx.attr.env_vars),
         ctx.attr.cmake_options,
+        inputs.include_dirs,
```

Comment:
Don't pass keyword arguments positionally (this is also wrong for the next argument, but at least do not make it worse).

Independently of this change, the limit for the number of positional arguments that is still managable seems to be exceeded for this function.